examples/asset/texture_sampler.rs

```diff
@@ -0,0 +1,109 @@
+//! By default Bevy loads images to textures with sampler settings that clamp the image to the edges.
+//! This example shows how to change the sampler settings to repeat the image instead.
+
+use bevy::app::App;
+use bevy::app::Startup;
+use bevy::asset::AssetServer;
+use bevy::asset::Assets;
+use bevy::math::Vec3;
+use bevy::prelude::*;
+use bevy::render::camera::ScalingMode;
+use bevy::render::mesh::Indices;
+use bevy::render::mesh::PrimitiveTopology;
+use bevy::render::texture::ImageAddressMode;
+use bevy::render::texture::ImageLoaderSettings;
+use bevy::render::texture::ImageSampler;
+use bevy::render::texture::ImageSamplerDescriptor;
+use bevy::sprite::MaterialMesh2dBundle;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_systems(Startup, setup)
+        .run();
+}
+
+/// Quad with UV coordinates in range `-1..=2`.
+fn quad_1_2() -> Mesh {
```

Comment:
I think the comment should explain _why_ it needs to be in the range `-1..=2`

Something like:

```suggestion
/// Quad with UVs that are larger than the quad to show the repeating texture.
fn quad() -> Mesh {
```

Reply:
UVs are not larger than quad, they are not comparable, like 2 kilograms is not larger than 1 kilometer. They are just outside of `0..=1` range. Which is the point of this example. Which is stated in the top of this file.

Reply:
Yeah, that comment is wrong I just meant a comment in the same style

Reply:
Sorry, I didn't get it. What's wrong with the style?